### ISSUE TITLE
Searching returns no result for some terms

### ISSUE THREAD

**Issue by hsluoyz** [algolia/docsearch#974]
**Do you want to request a _feature_ or report a _bug_?**

Bug

**What is the current behavior?**

Our site (https://casbin.org/) uses algolia's docsearch box: https://www.algolia.com/ref/docsearch/

But it seems to be not working for many search terms, like searching `codeigniter`:

![image](https://user-images.githubusercontent.com/3787410/91118819-72aadc00-e6c4-11ea-8fad-3983fd1fe6aa.png)

But clearly, we have such content:

![image](https://user-images.githubusercontent.com/3787410/91118864-86564280-e6c4-11ea-8195-f77e36a27a86.png)

Don't know why it can't be searched. I noticed this from our weekly report:

![image](https://user-images.githubusercontent.com/3787410/91118919-9cfc9980-e6c4-11ea-926f-1f3c59591e91.png)

Hope somebody can fix it.

**If the current behavior is a bug, please provide all the steps to reproduce and a minimal
[JSFiddle][3] example or a repository on GitHub that we can `npm install`
and `npm start`.**

Source code: https://github.com/casbin/casbin-website

**What is the expected behavior?**

Searching that term will return valid result.

[1]: https://github.com/algolia/docsearch-configs
[2]: https://docsearch.algolia.com/docs/config-file
[3]: https://jsfiddle.net/

**Comment by s-pace** [comment]
👋 

These terms are not searchable thanks to the changes on your configuration referenced above. The crawl has been run accordingly. You can see this on the following screenshot:
![image](https://user-images.githubusercontent.com/32097720/91147104-1641bf80-e6b8-11ea-8aba-426c506ee4e2.png)

Unfortunately, there is no way to redirect a user through tabs with DocSearch.

**Comment by hsluoyz** [comment]
@s-pace thanks for this quick fix!

I noticed another similar issue. In this page (https://casbin.org/docs/en/function#functions-in-matchers) we defined some built-in function names like `keyMatch3`, `keyMatch4`:

![image](https://user-images.githubusercontent.com/3787410/91147880-79991500-e6eb-11ea-9f28-f50937ec25a9.png)

But they are not searchable either. See: 

![image](https://user-images.githubusercontent.com/3787410/91147808-5e2e0a00-e6eb-11ea-8610-93fd951eb8fc.png)
